lib/office_search.rb

```diff
@@ -47,7 +47,7 @@ def self.find_exact_location(near)
   end
 
   def self.build_query_from_location(location, local_authority_id, opts)
-    q = Office.all
+    q = Office.where(office_type: :office)
```

Comment:
If offices can be offices and sometimes not, perhaps there's a better name for these..?

Reply:
yes, this is another victim of the data team changing the modelling of the data halfway through - I used the names they used originally for the concepts, but then they renamed Offices to Advice Locations (amongst other stuff) but I've not embarked on the process of renaming the table etc